test/posix-mock.h

```diff
@@ -39,6 +39,8 @@ typedef unsigned size_t;
 typedef int ssize_t;
 errno_t sopen_s(int* pfh, const char* filename, int oflag, int shflag,
                 int pmode);
+errno_t wsopen_s(int* pfh, const wchar_t* filename, int oflag, int shflag,
+                int pmode);
```

Comment:
I don't think this is needed because there is no corresponding test in posix-mock-test. You'll need to remove `FMT_POSIX_CALL` around the call to `wsopen_s` as well.